.github/workflows/integration_test.yaml

```diff
@@ -328,14 +324,15 @@ jobs:
         if: ${{ inputs.load-test-enabled }}
         run: k6 run load_tests/load-test.js ${{ inputs.load-test-run-args }}
       - name: Run Github Trivy FS Action
+        id: trivy-fs-action
         if: ${{ inputs.trivy-fs-enabled }}
         uses: aquasecurity/trivy-action@master
         with:
           scan-type: 'fs'
           scan-ref: ${{ inputs.trivy-fs-ref }}
           trivy-config: ${{ inputs.trivy-fs-config }}
       - name: Run Github Trivy Image Action
-        if: ${{ inputs.trivy-image-enabled && inputs.trivy-image-ref != '' }}
+        if: ${{ inputs.trivy-image-ref != '' }}
```

Comment:
Would it be better to run the image scan as the last step of the `build-images` job? That way there would be no need for parametrizing the image name

Reply:
Unfortunately, this parameter doesn't support multiple images, so I left it to be set.
There is a solution here but without using the action.
https://github.com/aquasecurity/trivy-action/issues/130

Reply:
The thing is that the `build-images` jobs is running in a matrix, so we would effectively be running this action once per image built.

Reply:
Nice! Thanks